package.json

```diff
@@ -16,6 +16,10 @@
     }
   },
   "dependencies": {
-    "atom-linter": "^4.5.1"
-  }
+    "atom-linter": "^4.5.1",
+    "atom-package-deps": "latest"
```

Comment:
Specify a version, `"latest"` can break very quickly on you if a major release comes out, this should be `"^4.0.1"`.